### PR TITLE
ARC-403: Update Installation instructions and add info about GitHub Actions for Jira

### PR DESCRIPTION
**Pull request by atrigueiro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,14 +30,27 @@ Connect your code with your project management in Jira. A separate Jira subscrip
...
-1. Sign into your Jira Cloud account
-2. Open the left sidebar by clicking on **Personal Settings**, if the left side bar is not already open. From the left sidebar in Jira, select **Jira Settings** -> **Apps** -> **Find new Apps**. (If you're using an older version of Jira, you won't have a left sidebar. Instead, click the **Gear Icon** in the top-right corner and select **Settings**. From there, select **Manage add-ons** from the left sidebar.)
-3. Search for **GitHub for Jira** and Click **Install**
-![image](https://user-images.githubusercontent.com/13207348/46588299-08550800-ca68-11e8-8ed4-290533320ef4.png)
-7. Click the **Get Started** button to connect your GitHub account.
 
-Next you will need to connect your GitHub organization to Jira, see the following steps.
+### Installations
+
+You can install this app from the Atlassian Marketplace, from your Jira instance and from the GitHub Marketplace. The instructions are below, you can choose whichever you prefer.
```
Comment (rachellerathbone) [inline README.md:36]: Should probably say 'or from the GitHub Marketplace'. And implies you need to do complete all options listed.

What's the difference between the Atlassian Marketplace and a Jira instance? Don't I need to be in a Jira instance to access the marketplace?

**Review by rachellerathbone — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (atrigueiro) [reply README.md:36]: No, you can access the Atlassian marketplace without even being logged in. If you decide to install the app, you'll then need to log in with your Atlassian account, and after you'd select the Jira instance you want to install the app. I've added some screenshots.

Comment (rachellerathbone) [reply README.md]: Oh interesting. Just another path we need to be mindful of. Sigh.

**Review by rachellerathbone — APPROVED** [review]

**Comment by mboudreau** [comment]
@atrigueiro You should really add a better description of your PR in the first comment of what you've changed.  Also, there's conflict with the merge I did yesterday that clarified Github Cloud vs Github Enterprise Server.

**Pull request merged**